commons/src/main/java/info/metadude/android/eventfahrplan/commons/SystemClock.kt

```diff
@@ -0,0 +1,112 @@
+package info.metadude.android.eventfahrplan.commons
+
+import org.threeten.bp.*
```

Comment:
# Imports
Please expand imports. Having each import separately facilitates easier conflict resolution while rebasing and any diff becomes more readable.
You might have to change your settings in Android Studio to do so. Check _Settings ... :arrow_right: Editor / Code Style / Kotlin / Imports_.

Reply:
Resolved in 23d7e05a3226ef2860144ab3fb7222db036a8fe0. Thanks.